### ISSUE TITLE
Modules and Plugins with Known Issues Doc Update 

### ISSUE THREAD

**Issue by tinefin** [pantheon-systems/documentation#2684]
Re: [Modules and Plugins with Known Issues](https://pantheon.io/docs/unsupported-modules-plugins/)

Priority (Low‚ Medium‚ High):

## Issue Description:
Add Query Monitor WP plugin to unsupported list

## Suggested Resolution 

**Query Monitor** - https://wordpress.org/plugins/query-monitor/
This plugin symlinks the `wp-content/db.php` file to a file within the plugin itself:
https://github.com/johnbillion/query-monitor#a-note-on-query-monitors-implementation

This symlink is set up with absolute path, which will only work on the exact appserver the plugin was installed on.
In short: The plugin is not designed for either a deployment workflow, cloud hosting, or multi server environment. 

Solution:
Pantheon has several tools in place to monitor database queries:
https://pantheon.io/docs/mysql-slow-log/
https://pantheon.io/docs/debug-mysql-new-relic/
